qa/distros/all/rhel_9.yaml

```diff
@@ -0,0 +1 @@
+rhel_9.3.yaml
```

Comment:
@kotreshhr ran into an issue - https://pulpito.ceph.com/khiremat-2024-01-19_06:49:40-fs:workload-wip-khiremat-49945-referent-inode-2-distro-default-smithi/7521650/

```
Error reimaging machines: Fog has no smithi_rhel_9.3 image. Available smithi images: ['smithi_centos_8.stream', 'smithi_centos_9.stream', 'smithi_rhel_8.2', 'smithi_rhel_8.4', 'smithi_rhel_8.6', 'smithi_ubuntu_16.04', 'smithi_ubuntu_18.04', 'smithi_ubuntu_20.04', 'smithi_ubuntu_22.04']
```

Temp fog issue or missing image?

Reply:
oops, I guess rhel9 is not in fog yet.

Reply:
We'll use centos9 instead!

Reply:
https://tracker.ceph.com/issues/64148

Reply:
Yeh.

@cbodley -- I can test the fs suite with this change along with #53901 once the changes are ready.